site/client/pages/Information/templates/About.html

```diff
@@ -12,7 +12,9 @@ <h2 class="primary-color">About EmpireNode</h2>
           EmpireNode is the premier one day, single-track, community run Node.js conference in NYC. Each year we highlight the cutting-edge of what the <i>fork</i> is going on with Node.js. The conference includes talks and workshops brought to you by some of the brightest minds in the community. Last year we covered everything from Gestural Color Spaces, to Cloud Automation in Node.
         </p>
 
-        <i class="lead">Join us again this year to learn what's new in Node in 2015, and to learn new skills you can apply on a daily basis. Check back soon for tickets!</i>
+        <i class="lead">Join us again this year to learn what's new in Node in 2015, and to learn new skills you can apply on a daily basis.</i>
```

Comment:
I think it would be nice to rephrase this paragraph to get rid of double-`learn`